pkg/snapstore/local_snapstore.go

```diff
@@ -58,6 +58,13 @@ func (s *LocalSnapStore) Save(snap brtypes.Snapshot, rc io.ReadCloser) error {
 	if err != nil && !os.IsExist(err) {
 		return err
 	}
+	if snap.IsChunk {
+		tok := strings.Split(snap.SnapName, "/")
+		err = os.MkdirAll(path.Join(s.prefix, snap.SnapDir, tok[0]), 0700)
```

Comment:
```suggestion
		err = os.MkdirAll(path.Join(s.prefix, snap.SnapDir, tok[0]), 0600)
```

Reply:
I think the permissions need to be 0700 only as I am creating directory to store chunks. So changing to 0600 doesn't allow me to do that through the code. It only allows me to create files, but not directories.

Reply:
ok, no worries.